tsnet/tsnet.go

```diff
@@ -182,7 +183,10 @@ func (s *Server) getAuthKey() string {
 	return os.Getenv("TS_AUTHKEY")
 }
 
-func (s *Server) start() error {
+func (s *Server) start() (reterr error) {
+	var closePool closeOnErrorPool
+	defer closePool.closeAllIfError(&reterr)
```

Comment:
so just to make sure I understand, if an error is returned, this. loop does not run

Reply:
Correct.